.github/workflows/test-pypi-publish.yml

```diff
@@ -0,0 +1,39 @@
+on:
```

Comment:
Once again just confirming that I've got the right idea here. This is just setup for Github workflows, right?

Reply:
Yes this is a new GHA workflow that is pretty much like the regular publish one except it triggers for version tags containing `rc`. If I understand what you are asking.